### DISCUSSION TITLE
Change search path while in Files

### DISCUSSION THREAD

**Discussion by dapc11** [ibhagwan/fzf-lua#1148] [Q&A] [answered]
Hello, thanks for a great plugin.

I've skimmed through the issue tracker and discussions, but can't find a topic that covers my question. Is there a way to change the cwd in an active session of Files?

**Comment by ibhagwan** [comment]
`:FzfLua files cwd=…` or `:lua require'fzf-lua'.files({ cwd = … })`.

**Reply by dapc11** [reply]
Sorry for being unclear. So I have already a "files" picker up and running. But realize that the file I'm after is located in the parent directory. So what I'm after is to be able to navigate the path within the file picker.

**Reply by ibhagwan** [reply, marked as answer]
You’d have to build that functionality yourself to go up one level in the tree by calling `files` again with the parent dir (using the actions table to bind it to a key of your choosing).

**Reply by ibhagwan** [reply]
Maybe this is what you’re after?
https://github.com/michel-garcia/fzf-lua-file-browser.nvim

**Comment by dapc11** [comment]
Will try to compile something myself, for my learnings sake. Thanks for the pointers and quick answers!

**Reply by ibhagwan** [reply]
Feel free to post your code once complete if you want me to review it
